src/components/TopBar/TopBar.js

```diff
@@ -514,6 +531,67 @@ const TopBar = ({
                                                 </ToggleButtonGroup>
                                             </StyledMenuItem>
                                         )}
+                                        {/*Languages */}
+                                        <StyledMenuItem
+                                            disabled={true}
+                                            style={{
+                                                opacity: '1',
```

Comment:
why opacity: '1' ?

Reply:
Here StyledMenuItem is disabled as we want to click on the toggle button. But we don't want the item to have disabled style (reduced opacity).